website/content/docs/setup/vault-configuration/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Vault Configuration
```

Comment:
We should reference Vault as the implementation details, not the concept.
How about using the term Credential Manager or credential store?
The text should reflect it as well.

Reply:
Sure sounds good. Will change it.